### DISCUSSION TITLE
issue with Loading livewire.js over HTTPS: net::ERR_HTTP2_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by DeltaCorporate** [livewire/livewire#7427] [1. Help]
### Description:
Encountering an error when accessing my Livewire application over HTTPS on Laravel 10 and Filament 3, hosted on OVH (Web Perso plan). The browser fails to load `livewire.js` with the error: `GET https://grafcet.net/livewire/livewire.js?id=2b77c128 net::ERR_HTTP2_PROTOCOL_ERROR 200 (OK)`. This issue is not present over HTTP.

### Steps to Reproduce:
1. Navigate to https://grafcet.net (Livewire application) over HTTPS.
2. Check the browser console for errors.

### Expected behavior:
The `livewire.js` file should load successfully without errors.

### Actual behavior:
The browser console shows `net::ERR_HTTP2_PROTOCOL_ERROR` when attempting to load `livewire.js`.

### Environment:
- Livewire version: 3.x
- Laravel version: 10
- Filament version: 3
- Browser: any browser
- Hosting: OVH Web Perso (mutualized hosting)
- Note: No shell access due to mutualized hosting environment.

### Additional context:
- The SSL certificate appears to be correctly configured as other resources load over HTTPS without issue.
- The issue seems specific to Livewire and HTTPS; no errors occur over HTTP.
- All publishable resources for Livewire have been published.
- The site URL: https://grafcet.net

I would appreciate any guidance or suggestions to resolve this issue.

**Comment by Pistil-Studio** [comment]
I, I have the same issue, your problem seems to be fixed ? what was your solution ? Thx

**Reply by kgsint** [reply]
@Pistil-Studio I might be wrong but maybe you need to configure [TrustProxies middleware](https://laravel.com/docs/10.x/requests#configuring-trusted-proxies)?

**Comment by pepe84** [comment]
I have same issue at OVH server. I fix it with this command (see #241):

`php artisan livewire:publish --assets`

I've read that you can automate the asset publishing adding this command in composer.json file:
```
...
    "scripts": {
        "post-autoload-dump": [
            ...
            "@php artisan livewire:publish --assets"
        ],
...
```

**Comment by DeltaCorporate** [comment]
Add the following PHP code to the file `vendor/livewire/src/Drawer/Utils.php` at line 75, before the return statement:

```php
return response()->make(
    content: file_get_contents($file),
    headers: $headers,
);
 but maybe the answer of @pepe84 is better
